node-fs-readfile/cat.ts

```diff
@@ -0,0 +1,17 @@
+import fs from 'node:fs/promises';
+
+async function readSelected(): Promise<void> {
+  try {
+    const inputs = process.argv.slice(2);
+    const concatFiles = inputs.map((file) => fs.readFile(`./${file}`, 'utf-8'));
```

Comment:
You don't need the `./` here. You can just pass `file` directly. That way you could type a full path into the command line and it would still work.